### PR TITLE
[Test Controller] New test controller applied to Automation

### PR DESCRIPTION
**Pull request by vladimir-shcherbakov**
## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [ ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by MiYanni** [comment]
@vladimir-shcherbakov There are merge conflicts.

**Comment by vladimir-shcherbakov** [comment]
@vrdmr We are applying a new test controller - could you take a look?

**Comment by vrdmr** [comment]
@sourabhguha, @GaneshMSAzure, @NarayanThiru, @kjohn-msft, @safeermohammed, @AnatoliB, @jemex, @NicoletazMS and @finiteattractor: FYI - New Test Controller for Automation cmdlets

**Comment by vladimir-shcherbakov** [comment]
signing: https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/psnetsign-esrp/116/

**Comment by vladimir-shcherbakov** [comment]
@MiYanni  Are you ready to sign off?

**Comment by MiYanni** [comment]
@vladimir-shcherbakov Please reassign to someone else. Thanks.

File: src/Automation/Automation.Test/ScenarioTests/VariableTests.cs [hunk 1]
```diff
@@ -12,20 +12,18 @@
 // limitations under the License.
 // ----------------------------------------------------------------------------------
 
+using Microsoft.Azure.Commands.Network.Test.ScenarioTests;
```
Comment (markcowl) [inline src/Automation/Automation.Test/ScenarioTests/VariableTests.cs]: same comment

File: src/Automation/Automation.Test/ScenarioTests/WebhookTests.cs [hunk 2]
```diff
@@ -12,61 +12,58 @@
 // limitations under the License.
 // ----------------------------------------------------------------------------------
 
+using Microsoft.Azure.Commands.Network.Test.ScenarioTests;
```
Comment (markcowl) [inline src/Automation/Automation.Test/ScenarioTests/WebhookTests.cs]: same

File: src/Automation/Automation.Test/ScenarioTests/HybridWorkerGroupTests.cs [hunk 3]
```diff
@@ -12,29 +12,26 @@
 // limitations under the License.
 // ----------------------------------------------------------------------------------
 
+using Microsoft.Azure.Commands.Network.Test.ScenarioTests;
```
Comment (markcowl) [inline src/Automation/Automation.Test/ScenarioTests/HybridWorkerGroupTests.cs]: same comment

File: src/Automation/Automation.Test/ScenarioTests/JobScheduleTests.cs [hunk 4]
```diff
@@ -12,25 +12,18 @@
 // limitations under the License.
 // ----------------------------------------------------------------------------------
 
+using Microsoft.Azure.Commands.Network.Test.ScenarioTests;
```
Comment (markcowl) [inline src/Automation/Automation.Test/ScenarioTests/JobScheduleTests.cs]: same comment

File: src/Automation/Automation.Test/ScenarioTests/ModuleTests.cs [hunk 5]
```diff
@@ -12,69 +12,66 @@
 // limitations under the License.
 // ----------------------------------------------------------------------------------
 
+using Microsoft.Azure.Commands.Network.Test.ScenarioTests;
```
Comment (markcowl) [inline src/Automation/Automation.Test/ScenarioTests/ModuleTests.cs]: same comment

File: src/Automation/Automation.Test/ScenarioTests/AccountTests.cs [hunk 6]
```diff
@@ -12,37 +12,33 @@
 // limitations under the License.
 // ----------------------------------------------------------------------------------
 
-using Microsoft.Azure.Commands.Automation.Test;
-using Microsoft.Azure.ServiceManagement.Common.Models;
+using Microsoft.Azure.Commands.Network.Test.ScenarioTests;
```
Comment (markcowl) [inline src/Automation/Automation.Test/ScenarioTests/AccountTests.cs]: Why is the using for the network scenario tests required?

File: src/Automation/Automation.Test/ScenarioTests/AutomationTestRunner.cs [hunk 7]
```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Azure.Commands.TestFx;
+using Microsoft.WindowsAzure.Commands.ScenarioTest;
+using Xunit.Abstractions;
+
+namespace Microsoft.Azure.Commands.Network.Test.ScenarioTests
```
Comment (markcowl) [inline src/Automation/Automation.Test/ScenarioTests/AutomationTestRunner.cs]: The namespace here should be Microsoft.Azure.Commands.Automation.Test or Commands.Automation.Test

File: src/Automation/Automation.Test/ScenarioTests/AutomationTests.cs [hunk 8]
```diff
@@ -12,84 +12,81 @@
 // limitations under the License.
 // ----------------------------------------------------------------------------------
 
-using Microsoft.Azure.ServiceManagement.Common.Models;
+using Microsoft.Azure.Commands.Network.Test.ScenarioTests;
```
Comment (markcowl) [inline src/Automation/Automation.Test/ScenarioTests/AutomationTests.cs]: see namepspace comment above

File: src/Automation/Automation.Test/ScenarioTests/CertificateTests.cs [hunk 9]
```diff
@@ -12,25 +12,18 @@
...
 namespace Commands.Automation.Test
 {
-    using System;
-    using System.Collections.Generic;
-    using System.Linq;
-    using System.Text;
-    using System.Threading.Tasks;
-    using Microsoft.Azure.Commands.Automation.Test;
-    using Microsoft.Azure.ServiceManagement.Common.Models;
+    using Microsoft.Azure.Commands.Network.Test.ScenarioTests;
```
Comment (markcowl) [inline src/Automation/Automation.Test/ScenarioTests/CertificateTests.cs]: same comment

File: src/Automation/Automation.Test/ScenarioTests/ConnectionTests.cs [hunk 10]
```diff
@@ -12,25 +12,18 @@
...
 namespace Commands.Automation.Test
 {
-    using System;
-    using System.Collections.Generic;
-    using System.Linq;
-    using System.Text;
-    using System.Threading.Tasks;
-    using Microsoft.Azure.Commands.Automation.Test;
-    using Microsoft.Azure.ServiceManagement.Common.Models;
+    using Microsoft.Azure.Commands.Network.Test.ScenarioTests;
```
Comment (markcowl) [inline src/Automation/Automation.Test/ScenarioTests/ConnectionTests.cs]: same comment, and for all those below

**Pull request merged**